src/main/java/edu/greenblitz/tobyDetermined/subsystems/LED.java

```diff
@@ -31,10 +31,9 @@ public static void init(){
         }
     }
 
-    public void setColor(Color color) {
+    public void setColor(Color color) { //to all
```

Comment:
If you need a comment to explain what the function does, it means the function name is not good enough. How about setEntireLedColor to something like that?

Reply:
I see now that you used overloading, so the name is good, If you want keep the comment, I think you don't have to have it.